### ISSUE TITLE
Sometimes failed to get metric

### ISSUE THREAD

**Issue by feli5** [hiveco/conntrack_exporter#3]
Below is the entire output of the exporter 
```
# HELP exposer_bytes_transferred bytesTransferred to metrics services
# TYPE exposer_bytes_transferred counter
exposer_bytes_transferred 165825104.000000
# HELP exposer_total_scrapes Number of times metrics were scraped
# TYPE exposer_total_scrapes counter
exposer_total_scrapes 961.000000
# HELP exposer_request_latencies Latencies of serving scrape requests, in milliseconds
# TYPE exposer_request_latencies histogram
exposer_request_latencies_count 961
exposer_request_latencies_sum 28441.000000
exposer_request_latencies_bucket{le="1.000000"} 633
exposer_request_latencies_bucket{le="5.000000"} 639
exposer_request_latencies_bucket{le="10.000000"} 651
exposer_request_latencies_bucket{le="20.000000"} 696
exposer_request_latencies_bucket{le="40.000000"} 772
exposer_request_latencies_bucket{le="80.000000"} 837
exposer_request_latencies_bucket{le="160.000000"} 933
exposer_request_latencies_bucket{le="320.000000"} 959
exposer_request_latencies_bucket{le="640.000000"} 959
exposer_request_latencies_bucket{le="1280.000000"} 959
exposer_request_latencies_bucket{le="2560.000000"} 959
exposer_request_latencies_bucket{le="+Inf"} 961
```

**Comment by dbendelman** [comment]
I'm guessing you ran it under docker? Which version of docker & conntrack_exporter and what was your commandline?

@feli5 please provide a sample of output from:

```
$ docker pull hiveco/conntrack_exporter
$ docker run --rm -it --cap-add=NET_ADMIN --net=host --name=conntrack_exporter hiveco/conntrack_exporter -d
```

As well the what the `/metrics` endpoint shows for the above.

**Comment by LERUfic** [comment]
this is also happening to me, I only get these metrics

```
# HELP exposer_transferred_bytes_total Transferred bytes to metrics services
# TYPE exposer_transferred_bytes_total counter
exposer_transferred_bytes_total 1160731
# HELP exposer_scrapes_total Number of times metrics were scraped
# TYPE exposer_scrapes_total counter
exposer_scrapes_total 519
# HELP exposer_request_latencies Latencies of serving scrape requests, in microseconds
# TYPE exposer_request_latencies summary
exposer_request_latencies_count 519
exposer_request_latencies_sum 776717
exposer_request_latencies{quantile="0.5"} 2713
exposer_request_latencies{quantile="0.9"} 3124
exposer_request_latencies{quantile="0.99"} 3492
```

Right now I'm using v0.3.1
In my case, I'm running it on my GCE VM

I don't know if it's related but in debug log I only see this warning.
```
[DEBUG] WARNING: Tried to update an existing connection in our table but a match was not found (rebuilding=false):
	event=update     ipv4     2 tcp      6 120 TIME_WAIT src=10.0.0.1 dst=10.0.0.2 sport=64841 dport=9318 src=10.0.0.2 dst=10.0.0.1 sport=9318 dport=64841 [ASSURED]
```
